### PR TITLE
Enrollment section updated

### PR DESCRIPTION
**Pull request by Vinyl-Davyl**

### CODE REVIEW

**Review by pablo-clueless — APPROVED** [review]

File: src/components/PaystackIntegration.jsx [hunk 1]
```diff
@@ -28,18 +59,19 @@ const PaystackIntegration = () => {
...
   const [courses, setCourses] = useState()
 
+
   useEffect(() => {
     const getCourse = async () => {
 
       try{
         const res = await axios.get(`${base_url}/course`)
         setCourses(res.data.data[0])
-        setAmount(res.data.data[0].price)
```
Comment (taiwoAkindele) [inline src/components/PaystackIntegration.jsx:42]: Why did you delete this? We are getting the price from here.

File: src/components/PaystackIntegration.jsx [hunk 2]
```diff
@@ -79,63 +109,247 @@ const PaystackIntegration = () => {
...
+                    <TextField
+                      label="First Name"
+                      type="text"
+                      value={firstName}
+                      onChange={(e) => setFirstName(e.target.value)}
+                    />
+                    <TextField
+                      label="LastName"
+                      type="text"
+                      value={firstName}
```
Comment (taiwoAkindele) [inline src/components/PaystackIntegration.jsx:287]: Value should be `LastName`

Comment (taiwoAkindele) [inline src/components/PaystackIntegration.jsx:287]: Value should be `LastName`

File: src/components/PaystackIntegration.jsx [hunk 3]
```diff
@@ -79,63 +109,247 @@ const PaystackIntegration = () => {
...
+                      onChange={(e) => setFirstName(e.target.value)}
```
Comment (taiwoAkindele) [inline src/components/PaystackIntegration.jsx:288]: onChange should be `setLastName(e.target.value)`

File: src/components/PaystackIntegration.jsx [hunk 4]
```diff
@@ -79,63 +109,247 @@ const PaystackIntegration = () => {
...
+                    <FormControl fullWidth>
+                      <InputLabel id="academic-qualification-label">Academic Qualification</InputLabel>
+                      <Select
+                        labelId="academic-qualification-label"
+                        id="academic-qualification"
+                        value={qualification}
+                        label="Academic Qualification"
+                        onChange={handleChange}
+                      >
+                        <MenuItem value={10}>Bachelor's Degree</MenuItem>
```
Comment (taiwoAkindele) [inline src/components/PaystackIntegration.jsx:325]: change `value` to something more distinct, this clashes with that of `academic qualifications` fields and one field will end up controlling the other.

**Review by taiwoAkindele — CHANGES_REQUESTED** [review]

File: src/components/PaystackIntegration.jsx (hunk 1 above)
Comment (Vinyl-Davyl) [reply src/components/PaystackIntegration.jsx:42]: Ohh, thought it wasn't in use.
Resolved!!

File: src/components/PaystackIntegration.jsx (hunk 3 above)
Comment (Vinyl-Davyl) [reply src/components/PaystackIntegration.jsx:288]: Yh 
Mistake Resolved!!

**Review by taiwoAkindele — APPROVED** [review]

**Pull request merged**